forms-web-app/src/validators/appellant-submission/appeal-statement-schema.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
           'Doc is the wrong file type: The file must be a DOC, DOCX, PDF, TIF, JPG or PNG'
         );
 
-        validateFileSize(size, config.fileUpload.maxFileSizeBytes);
```

Comment:
explains where that value was coming from - by using my own var names + intermingling with the express-fileupload config, I really made that hard to understand.